core/math/color.cpp

```diff
@@ -415,11 +415,8 @@ Color Color::named(const String &p_name, const Color &p_default) {
 int Color::find_named_color(const String &p_name) {
 	String name = p_name;
 	// Normalize name.
-	name = name.replace(" ", "");
-	name = name.replace("-", "");
-	name = name.replace("_", "");
-	name = name.replace("'", "");
-	name = name.replace(".", "");
+	static Vector<String> to_replace({ " ", "-", "_", "'", "." });
+	name = name.replace(to_replace, "");
```

Comment:
Since several of these are plain erasures of individual characters I wonder if we should add an erase method as well, or even instead, that takes a vector of `char32_t` instead, it would be faster and cleaner I'd say

Reply:
I'll see about writing a PR for these two options later